pkg/sql/codegen/tensorflow/codegen.go

```diff
@@ -308,6 +308,7 @@ func categorizeAttributes(trainStmt *ir.TrainStmt) (trainParams, validateParams,
 func deriveFeatureColumnCode(trainStmt *ir.TrainStmt) (featureColumnsCode []string, fieldDescs []*ir.FieldDesc, err error) {
 	perTargetFeatureColumnsCode := []string{}
 	for target, fcList := range trainStmt.Features {
+		perTargetFeatureColumnsCode = []string{}
```

Comment:
Was this a bug? if so, how about removing line 309 and change this line to `:=`?